test/test_models.py

```diff
@@ -163,15 +159,22 @@ def _test_detection_model(self, name, dev):
         def check_out(out):
             self.assertEqual(len(out), 1)
 
+            def compact(tensor):
+                size = tensor.size()
+                elements_per_sample = functools.reduce(operator.mul, size[1:], 1)
+                if elements_per_sample > 30:
+                    return compute_mean_std(tensor)
+                else:
+                    return subsample_tensor(tensor)
```

Comment:
Some tensors are too large to store on expected (for example masks). If the size of the input is over 30 elements per record, then we assert its mean and standard dev instead.